app/assets/stylesheets/govuk_publishing_components/components/_layout-super-navigation-header.scss

```diff
@@ -267,6 +284,14 @@ $after-button-padding-left: govuk-spacing(4);
   }
 }
 
+.gem-c-layout-super-navigation-header__navigation-top-toggle-button.gem-c-layout-super-navigation-header__navigation-top-toggle-button--blue-background {
+  &:focus:not(:focus-visible) {
+    &::after {
```

Comment:
There are a few places in the PR where we nest things quite deeply. I'm wondering if it might be possible to write the out the full class instead? The Design System coding standards which we mostly try to follow in the gem discourage the use of the ambersand and deep nesting: https://github.com/alphagov/govuk-frontend/blob/main/docs/contributing/coding-standards/css.md#nesting It might not be possible to do that in all the places but I think it would be good to do where possible to make the code easier to read.

Reply:
There is so much nesting going on already in this .scss that I think it's a whole other issue to refactor the entire stylesheet to get into line with the Design System coding standards. 

Personally I'm also not sure if it would make it easier to read as this line would then be:

`.gem-c-layout-super-navigation-header__navigation-top-toggle-button.gem-c-layout-super-navigation-header__navigation-top-toggle-button--blue-background:focus:not(:focus-visible)::after {`

which I myself don't find easier to read, I think the nesting makes more sense here.

Reply:
The gem docs address the issue of the long nested class names [here](https://github.com/alphagov/govuk_publishing_components/blob/main/docs/component_conventions.md#bem) with:
> To avoid long and complicated class names, we follow the [BEM guidance](http://getbem.com/faq/#css-nested-elements) that classes do not have to reflect the nested nature of the DOM.

But it's true that there is a lot of existing nesting in this component already so maybe it's not worth trying to overhaul the standard in this PR.

Reply:
I think if we were to do it in this instance then we would/should do it for all instances and it is 1000 lines of css. It's probably a task for when we don't have an imminent deadline.